cosmos-sdk-proto/src/lib.rs

```diff
@@ -5,8 +5,7 @@
 //! worth considering that modules seem to show up and go away with every RC.
 //! TODO actually implement features tag based compilation
 
-#![deny(warnings, trivial_casts, trivial_numeric_casts, unused_import_braces)]
-#![allow(clippy::large_enum_variant)]
```

Comment:
This clippy lint used to have an annoyingly small default but was increased to ~200 bytes somewhat recently, which I think is a reasonable threshold